examples/hello_flutter/BUILD.gn

```diff
@@ -4,7 +4,7 @@
 
 import("//flutter/build/flutter_app.gni")
 
-flutter_app("hello_flutter") {
+flutter_jit_app("hello_flutter") {
```

Comment:
Does this need to change? I thought most clients would use flutter_app and follow external configuration?